crypto/bn/bn_mul.c

```diff
@@ -525,14 +522,6 @@ void bn_mul_part_recursive(BN_ULONG *r, BN_ULONG *a, BN_ULONG *b, int n,
      * The zero case isn't yet implemented here. The speedup would probably
      * be negligible.
      */
-# if 0
-    if (n == 4) {
-        bn_mul_comba4(&(t[n2]), t, &(t[n]));
-        bn_mul_comba4(r, a, b);
```

Comment:
There actually are more #if-zeroed references to bn_mul_comba4. However! I used to use one of them to debug bn_mul_comba4. This in turn calls for question if even bn_mul_comba4 itself can be removed. At any case, I'd suggest to skip this removal from *this* request, so that issue can be looked at separately and from "wider angle". I can do this...